eras/dijkstra/impl/src/Cardano/Ledger/Dijkstra/UTxO.hs

```diff
@@ -43,7 +46,11 @@ instance EraUTxO DijkstraEra where
 
   getConsumedValue = getConsumedMaryValue
 
-  getProducedValue = conwayProducedValue
+  getProducedValue pp isRegPoolId txBody =
+    withBothTxLevels
+      txBody
+      (conwayProducedValue pp isRegPoolId)
```

Comment:
This call should be extracted into its own function and needs to be recursive on sub transactions, otherwise sub-transactions will not be included when balancing out the full transaction
```suggestion
      (conwayProducedValue pp isRegPoolId <> foldMap (getProducedValue pp isRegPoolId . (^. bodyTxL)) (txBody ^. subTxs)
```